apps/onnx/model.cpp

```diff
@@ -333,7 +333,7 @@ std::vector<py::array> run(
     }
     Halide::Realization real(outputs);
     Halide::Target tgt;
-    tgt.set_feature(Halide::Target::DisableLLVMLoopOpt);
+    tgt.set_feature(Halide::Target::DisableLLVMLoopUnroll);
```

Comment:
DisableLLVMLoopUnroll is not a Feature anymore; this could not possibly compile at top-of-tree. DisableLLVMLoopOpt is correct.

Reply:
You are correct. I updated my code locally, but I didn't rerun make test_apps from the top level so I ended up using the old version of the header files. This is fixed now.